### ISSUE TITLE
Someofselect, Selectcolumns, abstract-table, Fieldpicker controls should be accessible

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#481]
- Add aria-* attributes and roles to fix a11y violations in these controls -
- Add `role="presentation"` for all `<table>` elements without table headers.

**someofselect control -**
Remove "Scrollbar" column
![Screen Shot 2021-02-10 at 11 25 13 AM](https://user-images.githubusercontent.com/25124000/107560870-ba4df300-6b92-11eb-9ece-f058aad2093c.png)

**selectcolumns control -**
Remove "Scrollbar" column
![Screen Shot 2021-02-10 at 11 25 32 AM](https://user-images.githubusercontent.com/25124000/107560879-bd48e380-6b92-11eb-8755-deddb69d11e3.png)

**Abstract-table -**
- Remove "Scrollbar" column
- Add aria-label for abstract-table

**Fieldpicker -**
- Add aria-label for fieldpicker table

**Flexible-table -**
- Add label `"Search text field"` for seach bar in flexible table
